stubs/extensions/xdebug.phpstub

```diff
@@ -1,43 +1,66 @@
 <?php
 
-const XDEBUG_TRACE_APPEND = 1;
```

Comment:
Looks like they exist in [3.1.2](https://github.com/AndrolGenhald/php-extension-stub-generator/blob/master/generated_stubs/xdebug-3.1.2.phpstub) but not [3.0.0](https://github.com/AndrolGenhald/php-extension-stub-generator/blob/master/generated_stubs/xdebug-3.0.0.phpstub). I don't remember exactly, but I'm guessing I was thinking it'd be better to leave them out until we support `@since` for extensions. On the other hand, there's some functions in here with `@since 3.1`, so apparently I wasn't consistent in that.

Reply:
Mainly I'm worried about projects depending on those in any version. They're gonna have undefined constants if we remove that

Reply:
Yeah, I'll add it back and try to add support for `@since [extension-version]-x.y.z`. I think it's fine to show those undefined constant warnings for `"ext-xdebug": "*"` as long as it works correctly for `"ext-xdebug": "^3.1"`.